src/app/components/Table/TablePagination.tsx

```diff
@@ -5,13 +5,26 @@ import { useTranslation } from 'react-i18next'
 import { Link, To } from 'react-router-dom'
 
 export type TablePaginationProps = {
-  numberOfPages?: number
-  selectedPage: number
   linkToPage: (page: number) => To
+  rowsPerPage: number
+  selectedPage: number
+  offsetCount: number | undefined
+  isOffsetCountClipped?: boolean
 }
 
-export const TablePagination: FC<TablePaginationProps> = ({ numberOfPages, selectedPage, linkToPage }) => {
+export const TablePagination: FC<TablePaginationProps> = ({
+  selectedPage,
+  linkToPage,
+  rowsPerPage,
+  offsetCount,
+  isOffsetCountClipped,
+}) => {
   const { t } = useTranslation()
+  if (!offsetCount) {
+    return null
+  }
+  const totalCount = isOffsetCountClipped ? offsetCount : offsetCount + (selectedPage - 1) * rowsPerPage
```

Comment:
This doesn't completely fix new pages appearing as you navigate to last page. If a list contains between 1000 and 2000 items, then opening page 100 will reveal remaining pages.

I couldn't find a list with correct number of items, but this has 3611, so manually opening page 300 reveals remaining pages:
https://527cb3e5.oasis-explorer.pages.dev/emerald/account/oasis1qrmtfytxk3exv60rpkv8p8ve99sf3kzp8u9ng4gq?page=300

Options:
- show 100 pages + reveal remaining <99 (**current**)
- show 100 pages (_I think we would need to maintain hardcoded maxTotalCount that matches backend_)
- show 100 pages + reveal next 100 + reveal next 100 + reveal next 100 .. (_without this condition_)

Reply:
Ok I will revert last commit